canonical-websites/snapcraft-flask
# A basic Flask application

Just as a starting point, create a basic hello world Flask application.

QA
--

``` bash
FLASK_APP=hello.py flask run
```

Browse to http://127.0.0.1:5000/ and check you see "Hello, World!".
